File: 240111/함수를 이용한 부분 문자열의 위치 구하기/find-the-location-of-a-substring-using-a-function.java
import java.util.Scanner;
public class Main {
    public static int word3(String word, String word2){
        int cnt = 0 ; 
        for (int i = 0 ; i<=word.length()-word2.length(); i++){
            cnt = 0 ;
            for(int j = 0 ; j<word2.length(); j++){
                if (word.charAt(i+j) == word2.charAt(j)){
                    cnt++;
                }
            }
            if(cnt == word2.length()){
                return i;
            }
        }
        return -1;
        
    }
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        String word = sc.next();
        String word2 = sc.next();

        System.out.print(word3(word,word2));
    }
}

File: 240111/함수를 이용한 부분 문자열의 위치 구하기/find-the-location-of-a-substring-using-a-function.py
word = input()

word2 = input()

def ind(word,word2):
    cnt = 0 
    for i in range(len(word)-len(word2)):
        cnt = 0 
        for j in range(len(word2)):
            if word[i+j] == word[j]:
                cnt+=1 
        if cnt == len(word):
            return i
    return - 1

print(ind(word,word2))